python/sglang/srt/managers/schedule_batch.py

```diff
@@ -1168,8 +1171,10 @@ def merge_batch(self, other: "ScheduleBatch"):
 
     def get_model_worker_batch(self):
         if self.forward_mode.is_decode_or_idle():
+            decode_seq_lens = self.seq_lens.cpu()
```

Comment:
This will slowdown other things (e.g., speculative decoding where overlap scheduler is turned off). Can we only do this when needed?